java/opendb-api/build.gradle

```diff
@@ -77,12 +85,17 @@ dependencies {
     implementation group: 'org.bouncycastle', name: 'bcprov-jdk15on', version: '1.60'
 	implementation group: 'org.postgresql', name: 'postgresql', version: '9.4.1212'
 
-
+    compile group: 'commons-logging', name: 'commons-logging', version: '1.1.1'
+    compile group: 'org.springframework', name: 'spring-beans', version: '5.1.7.RELEASE'
```

Comment:
This dependency not compatible with neither:
spring-boot-... :2.0.1.RELEASE
:spring-security-...:5.0.6.RELEASE